src/Microsoft.Health.Fhir.Stu3.Core/Microsoft.Health.Fhir.Stu3.Core.csproj

```diff
@@ -25,7 +25,7 @@
     <PackageReference Include="Microsoft.Extensions.Http" />
     <PackageReference Include="Microsoft.Extensions.Logging" />
     <PackageReference Include="Hl7.Fhir.STU3" />
-    <PackageReference Include="Hl7.Fhir.Specification.STU3" PrivateAssets="build;analyzers"/>
+    <PackageReference Include="Hl7.Fhir.Specification.STU3" PrivateAssets="build;analyzers" />
```

Comment:
NIT: Empty space

Reply:
Hello! I'll keep this empty line, because the other lines in the file follow the same standard.